### PR TITLE
doc(gry): add mdqn doc

### PR DESCRIPTION
**Pull request by ruoyuGao**
- add mdqn doc 
- fix small typo in acrobot zh doc

### CODE REVIEW

File: source/12_policies/mdqn.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+-----------
+
+
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+| environment         |best mean reward | evaluation results                                  | config link              | comparison           |
++=====================+=================+=====================================================+==========================+======================+
+|                     |                 |                                                     |`config_link_p <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |  sdqn(3513)          |
+|                     |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|Asterix              |  8963           |.. image:: images/benchmark/mdqn_asterix.png         |atari/config/serial/      |  paper(1718)         |
```
Comment (PaParaZz1) [inline source/12_policies/mdqn.rst:88]: 1718 in paper?

File: source/12_policies/mdqn.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_s <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |  sdqn(986.1)         |
+|Enduro               |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|                     |  1003           |.. image:: images/benchmark/mdqn_enduro.png          |atari/config/serial/      |  paper(1171)         |
+|(Enduro-v4)          |                 |                                                     |endruo/space              |                      |
+|                     |                 |                                                     |enduro_mdqn_config.py>`_  |  dqn(986.4)          |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+
+
```
Comment (PaParaZz1) [inline source/12_policies/mdqn.rst:107]: add some naive explanation about the differences between and our config and paper config

File: source/12_policies/mdqn.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+.. autoclass:: ding.rl_utils.td.m_q_1step_td_error
+   :members: forward
+   :noindex:
+
+
+Benchmark
+-----------
+
+
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
```
Comment (PaParaZz1) [inline source/12_policies/mdqn.rst:82]: There is a more simple table format in rst, you can refer to this [implementation](https://raw.githubusercontent.com/opendilab/DI-engine-docs/main/source/12_policies/c51.rst).

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: source/12_policies/mdqn.rst (hunk 2 above)
Comment (ruoyuGao) [reply source/12_policies/mdqn.rst]: added

File: source/12_policies/mdqn.rst (hunk 3 above)
Comment (ruoyuGao) [reply source/12_policies/mdqn.rst]: changed

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**